google/cloud/compute/firewall_policies/v1/internal/firewall_policies_rest_connection_impl.cc

```diff
@@ -92,14 +92,10 @@ FirewallPoliciesRestConnectionImpl::AddAssociation(
       },
       [request](std::string const& op,
                 google::cloud::cpp::compute::global_organization_operations::
-                    v1::GetOperationRequest& r) {
-        r.set_operation(op);
-      },
+                    v1::GetOperationRequest& r) { r.set_operation(op); },
```

Comment:
nit: something went wrong in your sequence of commits (why change compute?). But the final result looks Okay. So 🤷

Reply:
Yes something did go wrong, I ran checkers twice I think... and added it as a fixup commit

Reply:
Ahh ... I guess this negates my previous comment.